lib/ansible/modules/clustering/kubernetes.py

```diff
@@ -325,6 +326,46 @@ def k8s_update_resource(module, url, data, patch_operation):
     return True, body
 
 
+def getVersion(module, kind, body):
+    if not kind:
+        module.fail_json(msg="Invalid kind '%s' for resource version" % kind)
+        return None
+    if kind == 'ConfigMap':
```

Comment:
Secret also seems to have `resourceVersion`.

https://github.com/kubernetes/kubernetes/blob/44f24d219f9d28121eb1f16863312f790e144882/staging/src/k8s.io/api/core/v1/types.go#L4689-L4692

Could you add it here?
I also need this PR but I want to apply this to Secret.

Reply:
Yes, no problem, I'll change my PR today to add Secret.

Reply:
thx!

Reply:
Updated the PR with "Secret" and "Service" as well. Build is passing.